addons/crm/models/crm_team.py

```diff
@@ -286,15 +252,16 @@ def action_assign_leads(self, work_days=1, log=True):
             }
         }
 
-    def _action_assign_leads(self, work_days=1):
+    def _action_assign_leads(self, force_quota=False):
```

Comment:
Just to be sure, as we understood it is already done on our prod.
So does `force_quota` was kept in the end ? We don't really have an issue with (the use case kind of make sense) or without (the code is bit simpler) it, just wanted to be sure if we keep it or not.